controls/roles/validator-import/tasks/main.yml

```diff
@@ -0,0 +1,176 @@
+
+  - name: Read service's configuration
+    slurp:
+      src: "{{ stereum_service_config_path }}"
+    register: stereum_service_configuration_raw
+    become: yes
+
+  - name: Read variable
+    set_fact:
+      stereum_service_configuration: "{{ stereum_service_configuration_raw.content | b64decode | from_yaml }}"
+    become: yes
+
+  - name: Build service container's name
+    set_fact:
+      stereum_service_container_name: "stereum-{{ stereum_service_configuration.id }}"
+
+  - name: Set directory permissions
+    file:
+      path: "{{ item.split(':') | first }}"
+      state: directory
+      owner: "{{ stereum_service_configuration.user }}"
+      group: "2000"
+      mode: 0700
+    become: yes
+    when: stereum.manage_service.state == "start" or stereum.manage_service.state == "restarted"
+    with_items: "{{ stereum_service_configuration.volumes }}"
+
+  - name: validator Stereum's docker network
+    docker_network:
+      name: stereum  
+
+  - name: Create validator keys
+    copy:
+      dest: "/opt/app/services/2e83d610-48b8-11ec-802a-330dee8e6b2e/launchpad/{{ item.name }}"
+      content: "{{ item.content }}"
+      owner: "{{ stereum_service_configuration.user }}"
+      group: "2000"
+      mode: 0700
+    become: yes
+    with_items: "{{ validator_keys }}"  
+
+  - name: Creating expect script for import 
+    copy:
+      dest: "/opt/app/import/lighthouse_validator_import.sh"
+      content: |
+        #!/usr/bin/expect
+
+        set timeout -1
+        set validator_password [lindex $argv 0];
+
+        if {[ llength $validator_password ] == 0 } {
+           exit
+        }
+
+        spawn sh -c { sudo docker run --rm -it \
+                       -e DEBUG_LEVEL=debug \
+                       -e NETWORK=prater \
+                       -e DATADIR="/opt/app/validator" \
+                       -e DIRECTORY="/opt/app/launchpad" \
+                       -v "/opt/app/services/2e83d610-48b8-11ec-802a-330dee8e6b2e/validator":"/opt/app/validator" \
+                       -v "/opt/app/services/2e83d610-48b8-11ec-802a-330dee8e6b2e/launchpad":"/opt/app/launchpad" \
+                       --user "2000":"2000" \
+                       --net "stereum" \
+                       --entrypoint "/opt/app/start/import-launchpad-wallet.sh" \
+                       validator_image
+                    }
+
+        expect {
+            "Enter the keystore password" {
+              send "$validator_password\r"
+              sleep 3
+              set results $expect_out(buffer)
+              if {[regexp -nocase "Invalid password" $results]} {
+                exit
+              } elseif {[regexp -nocase "Skipping import of keystore for existing public key" $results]} {
+                exit
+              }
+              exp_continue
+            }
+        } 
+      owner: "{{ stereum_service_configuration.user }}"
+      group: "2000"     
+      mode: 0700
+    become: yes   
+
+  - name: Start beacon service
```

Comment:
can you put this into `converge.yaml` please? when we run this role the beacon and validator should already run

you can make use of the "manage-service" role and call with parameters:
https://github.com/stereum-dev/ethereum-node/blob/main/controls/roles/manage-service/molecule/start-multiple/converge.yml#L10

Reply:
done!